lib/broccoli/ember-app.js

```diff
@@ -1072,12 +1072,26 @@ class EmberApp {
    * @return {BroccoliTree}
   */
   getAddonStyles() {
-    let mergedAddonStyles = mergeTrees(this.addonTreesFor('styles'), {
+    let addons = this.addonTreesFor('styles');
+
+    let flatten = addons.map(tree =>
+      new Funnel(tree, {
```

Comment:
Can you add comments in here about stuff to deprecate as people are doing things we don't want?